lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -142,6 +140,10 @@ public String apply(HeldLocksToken from) {
             this.locks = locks;
         }
 
+        public List<LockDescriptor> getLockDescriptors() {
+            return locks.stream().map(ClientAwareReadWriteLock::getDescriptor).collect(Collectors.toList());
```

Comment:
changed this piece, as it is in LockServiceImpl as this file generally is very performance sensitive. Leaving it in logger.